.env.development

```
@@ -0,0 +1,144 @@
+
+# 🏗️  APPLICATION CONFIGURATION
+NODE_ENV=development
+PORT=5000
+HOST=0.0.0.0
+APP_NAME=SMC_Trading_Agent
+APP_VERSION=1.0.0
+APP_DEBUG=true
+
+# 🔐  SECURITY & AUTHENTICATION
+JWT_SECRET=dev_jwt_secret_minimum_32_characters_long_for_security
+JWT_ACCESS_SECRET=dev_jwt_access_secret_minimum_32_characters_long_for_security
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_ACCESS_SECRET key should go before the JWT_SECRET key

(UnorderedKey)